torch/onnx/_internal/exporter/_ir_passes.py

```diff
@@ -90,7 +90,9 @@ def rename_axis(model: ir.Model, rename_mapping: dict[str, str]) -> None:
             value.shape = ir.Shape(new_shape)
 
 
-def add_torchlib_common_imports(model: ir.Model) -> None:
+def add_torchlib_common_imports(
```

Comment:
I assume this is used in other places as well

Reply:
here: https://github.com/pytorch/pytorch/blob/17f9276e29477f7a24bf431fa905f8bdd1464cb3/torch/onnx/_internal/exporter/_core.py#L1158

Reply:
but maybe the version converter handles it there.

Reply:
Yes, the version converter handles it here.